tests/src/Functional/GroupSubscribeTest.php

```diff
@@ -270,7 +270,7 @@ public function testSubscribeAccess() {
         // @todo This currently returns a 500 error due to a bug in core. Change
         //   this to a 403 or 404 when the bug is fixed.
         // @see https://www.drupal.org/node/2786897
-        'code' => 500,
+        'code' => version_compare(\Drupal::VERSION, '9.1', '>=') ? 404 : 500,
```

Comment:
Looks like this has been solved in [9.1.4](https://www.drupal.org/project/drupal/releases/9.1.4)
```suggestion
        'code' => version_compare(\Drupal::VERSION, '9.1.4', '>=') ? 404 : 500,
```